package.json

```diff
@@ -13,12 +13,15 @@
     "probot-app"
   ],
   "scripts": {
-    "dev": "nodemon",
-    "start": "probot run ./src/bot.js",
+    "dev": "npm run build:watch",
+    "start": "probot run ./dist/bot.js",
     "lint": "standard --fix",
-    "test": "jest && standard",
+    "jest": "jest --detectOpenHandles",
+    "test": "tsc && jest && standard",
     "test:watch": "jest --watch --notify --notifyMode=change --coverage",
-    "build": "webpack"
+    "build": "webpack",
```

Comment:
I'm not sure you can run webpack on it's own. The docs seem to suggest running it with `serverless` https://github.com/serverless-heaven/serverless-webpack#bundle-with-webpack.
```suggestion
    "build": "serverless webpack",
```

I ran it but got all kinds of environment variable errors so I'm not sure if it will succeed but hopefully it gets us farther?

Reply:
Yes, this did help me resolve my `yarn build problem` but I now get the same error I was getting before when I manually changed the entry.
![image](https://user-images.githubusercontent.com/1080613/91491572-40df7400-e882-11ea-80a8-cdd4984b1174.png)
src/bot.ts is a file, I wonder if I should `require('./bot.ts')` instead.